README.md

```diff
@@ -1,80 +1,190 @@
 
-5. Here is a fun little step. Try build a helper to set the profile picture. This step is optional (if you are satisfied with the default dull gray anonymous user image)
-Hint: Use the `image` collection for uploading the profile pictures. We have provided some of the code, but this [tutorial](https://medium.com/@victorleungtw/how-to-upload-files-with-meteor-js-7b8e811510fa) might be helpful in order to understand it. 
+We have exposed the *Instagram* object on the global *Tweeteor* object. All you will need is to use the *getImage* function to get an image representing a single tag. 
 
-### 3. Posts
-Oh, yes! This is the big one, here is where the fun starts.
+The Instagram fetcher must run on the server only.
 
-You are going to implement posts, all of saving, adding, and exctracting tags.
-You are free to implement as you wish.
-We have given you a head start by providing the styling and template files.
+*Tip: Start by creating one or two (or even more?) collections for you posts. Storing tags in a separate collection might not be a bad idea. Remember to subscribe and wait!*
 
-Hints:
-- **tag.html** & **tag.js** are just a proposal, you are free to make your own or customize it with your own structure.
-- Fetching tags from instagram MUST TO BE ON THE SERVER, so the method for adding a post should at least have an `if(isServer)` or `if(isClient) { return; }`
+*Tip: Underscore is available by default on the global scope, so if you need to do some filtering and such, feel free to use it.*
 
+*Tip: Get simple posts working first. No need to involve Instagram in the beginning. But when you do, it might be a good idea to drop all posts in your database to start with fresh data.*
 
-### 4. Bonus
+*Tip: You will need to extract tags from tweets. If you have a *#hashtag* you can use the following code to strip away anything besides the text: `str.replace(/[^A-Za-z0-9æøåÆØÅ]*/gi, '')`. Running a string '#hashtag.' thorugh this gives 'hashtag' as output.
 
-Here are some bonus tasks that are not in the solution branch:
+*Tip: *To make the username under each post clickable, there is a helper supplied by Iron router called *linkTo* which is very useful*
 
-1. Enable editing and deleting of posts.
+*Tip: To enable highlighting of hashtags on hoover on the images, you might find the package reactive-var useful, but not neccessarily! This task may prove more difficult than it sounds by the way*
 
-2. Make a tag-cloud and bundle it in a Meteor pacakage.
+*Tip: Remember that Meteor collections are reactive data sources. Anything in the DOM that has something to do with a collection, will probalby be easiest to solve using the collection somehow.*
```

Comment:
Anything in the DOM that should render automatically will probably be easiest to solve using collections.